app/renderer/components/preferences/syncTab.js

```diff
@@ -137,7 +137,7 @@ class SyncTab extends ImmutableComponent {
     return devices.map((device, id) => [
       {
         html: id,
-        value: parseInt(id)
+        value: device.get('lastRecordTimestamp')
```

Comment:
i think this could often be out-of-order with the ID. it's probably better to just pad the ID or change `value` to a string of 0's repeating `id` times.